lib/librte_regexdev/rte_regexdev_driver.h

```diff
@@ -0,0 +1,54 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(c) 2020 Mellanox Corporation
+ */
+
+#ifndef _RTE_REGEXDEV_DRIVER_H_
+#define _RTE_REGEXDEV_DRIVER_H_
+
+/**
+ * @file
+ *
+ * RTE RegEx Device PMD API
+ *
+ * These APIs for the use from regex drivers, user applications shouldn't
+ * use them.
+ *
```

Comment:
extra line or it should be consistent with line 34?
don't know yet.

Reply:
Will fix